frontend/packages/pipelines-plugin/src/components/taskruns/TaskRunDetailsSection.tsx

```diff
@@ -18,14 +19,14 @@ const TaskRunDetailsSection: React.FC<TaskRunDetailsSectionProps> = ({ taskRun }
           taskRunLabel: t(TaskRunModel.labelKey),
         })}
       />
-      <div className="row">
-        <div className="col-sm-6">
+      <Grid hasGutter>
+        <GridItem sm={6}>
           <ResourceSummary resource={taskRun} />
-        </div>
-        <div className="col-sm-6 odc-taskrun-details__status">
+        </GridItem>
+        <GridItem sm={6} className="odc-taskrun-details__status">
           <TaskRunDetailsStatus taskRun={taskRun} />
-        </div>
-      </div>
+        </GridItem>
+      </Grid>
```

Comment:
TestRun details also seems have issue
<img width="1458" alt="Screenshot 2025-05-06 at 5 21 15 PM" src="https://github.com/user-attachments/assets/5ddc70d3-a82f-4830-8ce3-a55fb913fafb" />

Reply:
Unable to reproduce:

![image](https://github.com/user-attachments/assets/1349cddb-7db6-4b04-8cb5-c5291acbd349)

Reply:
I'm also unable to reproduce today, sorry for the confusion